### ISSUE TITLE
Fine tuning for new topic

### ISSUE THREAD

**Issue by matbee-eth** [aigc-apps/EasyAnimate#24]
Would I be correct in saying that for fine-tuning a topic that the models were not trained on, I could simply finetune PixArt on the subject?

**Comment by zouxinyi0625** [comment]
yes, and also you can refer to train_t2iv.sh to train the whole model.
